db/oracle/docker/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM container-registry.oracle.com/database/free:latest
+# FROM container-registry.oracle.com/database/free:latest
```

Comment:
Tried several different versions.  The matching version is 19.3.0.0 and it imported the data with the least issues.